src/CodeAnalysis.ruleset

```diff
@@ -38,4 +38,47 @@
     <!-- Disable analyzer warning 1821 until https://github.com/dotnet/roslyn-analyzers/issues/1804 is resolved -->
     <Rule Id="CA1821" Action="None" />
   </Rules>
+  <Rules AnalyzerId="xunit.analyzers" RuleNamespace="xunit.analyzers">
+    <Rule Id="xUnit1004" Action="Hidden" />
+    <Rule Id="xUnit1005" Action="None" />
+    <Rule Id="xUnit1006" Action="None" />
+    <Rule Id="xUnit1007" Action="None" />
+    <Rule Id="xUnit1008" Action="None" />
+    <Rule Id="xUnit1009" Action="None" />
+    <Rule Id="xUnit1010" Action="None" />
+    <Rule Id="xUnit1011" Action="None" />
+    <Rule Id="xUnit1012" Action="None" />
+    <Rule Id="xUnit1013" Action="Error" />
```

Comment:
💡 Use warnings-as-errors to escalate warnings to errors.

```suggestion
    <Rule Id="xUnit1013" Action="Warning" />
```

Reply:
You mean basically for all rules? As we enabled warningsAsErrors=true it doesn't matter much.

Reply:
OK got what you said. applied, thanks.